pager.c

```diff
@@ -2265,27 +2271,35 @@ static void pager_custom_redraw(struct Menu *pager_menu)
  */
 int mutt_pager(const char *banner, const char *fname, PagerFlags flags, struct Pager *extra)
 {
+
+  struct Mailbox *m;
   static char searchbuf[256] = { 0 };
   char buf[1024];
   int ch = 0, rc = -1;
   bool first = true;
   int searchctx = 0;
   bool wrapped = false;
-
   struct Menu *pager_menu = NULL;
+  int msg_in_pager = 0;
```

Comment:
This has become orphaned.
It's set in a few places, but not read.

Rather than using this local, I think we should be using `Context->msg_in_pager`.
There are three places where it's set/used.